modules/RouteUtils.js

```diff
@@ -96,3 +100,38 @@ export function createRoutes(routes) {
 
   return routes
 }
+
+/**
+ * Checks a Route for the component that SHOULD be rendered against the
+ * component that WILL be rendered. If falsy, this indicates an
+ * unreachable route.
+ */
+function verifyRouteIsReachable(path, component, transitionManager) {
+
+  // strip vars so the path becomes routable
+  const cleanPath = path.replace(':', '')
```

Comment:
Yes, I think you will have to, as they cause a good percentage of the reachability issues.